ch12/handle_segv.c

```diff
@@ -39,6 +40,8 @@ typedef long unsigned int u64;
 /*---------------- Macros -------------------------------------------*/
 #define APPNAME "handle_segv"
 
+#define SIGNAL_STACK_SIZE 8192 /* Change this as per the signal handler*/
```

Comment:
Would prefer the alt stack size to be:
a) larger, say 8 MB, and
b) size should be depending on whether we're on a 32 or 64-bit arch (you can use the '#if __x86_64__ ' / else clause to do this programatically